detox/android/detox/src/test/java/com/wix/detox/DetoxActionHandlersSpec.kt

```diff
@@ -142,5 +142,82 @@ object DetoxActionHandlersSpec: Spek({
                         eq(messageId))
             }
         }
+
+        describe("Recording state actions") {
```

Comment:
This is good, but need to also add a use case checking that done is sent back when path isn't available.

Reply:
Also, what happens if params is null or malformed (i.e. invalid json)? Better cover that as well (expect as exception)

Reply:
Agree